server.js

```diff
@@ -46,6 +47,12 @@ sio.on('connection', function (socket) {
     socket.visibility = state;
     sio.emit('users:visibility-states', getVisibilityCounts());
   });
+  
+  socket.on('user-merge', function (edit,filepath) {
+	  fs.writeFileSync(filepath, edit.content,'utf8');
```

Comment:
je l'avais fait, j'aurais du le lancer en commentaire

/*function onInitFsWrite(fs) {
			//fs.root.getFile('‪C:\Users\Romain\Desktop\filesync-master\test\test3.txt', {create: true}, writeFile, errorHandler);
			fs.root.getFile(edit.filepath, {create: true}, writeFile, errorHandler);
			console.log('Systeme de fichier: ' + fs.name);
		}
		function onInitFsRead(fs) {
			//fs.root.getFile('‪C:\Users\Romain\Desktop\filesync-master\test\test3.txt', {create: true}, readFile, errorHandler);
			fs.root.getFile(edit.filepath, {create: true}, readFile, errorHandler);
			console.log('Systeme de fichier: ' + fs.name);
		}
		function onInitFsDelete(fs) {
			//fs.root.getFile('‪C:\Users\Romain\Desktop\filesync-master\test\test3.txt', {create: false}, deleteFile, errorHandler);
			fs.root.getFile(edit.filepath, {create: false}, deleteFile, errorHandler);
			console.log('Systeme de fichier: ' + fs.name);
		}
		function readFile(fileEntry) {
			fileEntry.file(function(file) {
				var reader = new FileReader();
				reader.onloadend = function(e) {
					alert(this.result);
				};
				reader.readAsText(file);
			}, errorHandler);
		}
		function deleteFile(fileEntry) {
			fileEntry.remove(function() {
				alert('delete OK');
			}, errorHandler);
		}		
		function writeFile(fileEntry) {
			fileEntry.createWriter(function(fileWriter) {
				fileWriter.onwriteend = function(e) {
					alert('ecriture OK.');
				};
				fileWriter.onerror = function(e) {
					alert('Error: ' + e.toString());
				};
				var data = new Blob([edit.content], {type: 'text/plain'});
				fileWriter.write(data);
			}, errorHandler);
		}
		function errorHandler(e) {
			var msg = '';
			switch (e.code) {
				case FileError.QUOTA_EXCEEDED_ERR:
					msg = 'QUOTA_EXCEEDED_ERR';
					break;
				case FileError.NOT_FOUND_ERR:
					msg = 'NOT_FOUND_ERR';
					break;
				case FileError.SECURITY_ERR:
					msg = 'SECURITY_ERR';
					break;
				case FileError.INVALID_MODIFICATION_ERR:
					msg = 'INVALID_MODIFICATION_ERR';
					break;
				case FileError.INVALID_STATE_ERR:
					msg = 'INVALID_STATE_ERR';
					break;
				default:
					msg = 'Unknown Error';
					break;
			};
			alert('Error: ' + msg);
		}
		window.requestFileSystem  = window.requestFileSystem || window.webkitRequestFileSystem;
		//window.requestFileSystem(window.TEMPORARY, 1024*1024, onInitFsDelete, errorHandler);
		window.requestFileSystem(window.TEMPORARY, 1024*1024, onInitFsWrite, errorHandler);
		window.requestFileSystem(window.TEMPORARY, 1024*1024, onInitFsRead, errorHandler); */